day3OfJS/day3.js

```diff
@@ -0,0 +1,356 @@
+*/
+const firstName = 'Luisa';
+const lastName = 'Ramírez';
+const country = 'Colombia';
+const city = 'Medellín';
+const age = 22;
+const isMarried = false;
+const year = 2021;
+
+console.log(typeof 'Luisa'); //string
+console.log(typeof 'Ramírez'); //string
+console.log(typeof 'Colombia'); //string
+console.log(typeof 'Medellín'); //string
+console.log(typeof 22); //number
+console.log(typeof false); //boolean
+console.log(typeof 2021); //number
+
+
+//2. Check if type of '10' is equal to 10
+const numberTenTypeofString = '10';
+const numberTenTypeofNumber = 10;
+console.log(numberTenTypeofString === numberTenTypeofNumber); //false
+
+
+//3. Check if parseInt('9.8') is equal to 10
+const numberFloatString = parseFloat('9.81');
+const theNumberTenTypeofNumber = 10
+console.log(numberFloatString === theNumberTenTypeofNumber); //false
+
+/*
+4. Boolean value is either true or false.
+i. Write three JavaScript statement which provide truthy value.
+*/
+const theStringTrue = true;
+const tenIsHigherThanNine = 10 > 9;
+const equalNumber = 1 === 1;
+
+console.log(theStringTrue); //true
+console.log(tenIsHigherThanNine); //true
+console.log(equalNumber); //true
+
+//ii. Write three JavaScript statement which provide falsy value.
+const emptyString = '';
+const tenIsHigherThanEleven = 10 > 11;
+const theEqualNumber = 1 === 2;
+
+console.log(emptyString); //false
+console.log(tenIsHigherThanEleven); //false
+console.log(theEqualNumber); //false
+
+
+/*
+5. Figure out the result of the following comparison expression first without using console.log(). After you decide the result confirm it using console.log()
+i. 4 > 3      //true
+ii. 4 >= 3    //true
+iii. 4 < 3    //false
+iv. 4 <= 3    //true
+v. 4 == 4     //true
+vi. 4 === 4   //true
+vii. 4 != 4   //false
+viii. 4 !== 4 //false
+ix. 4 != '4'  //true
+x. 4 == '4'   //true
+xi. 4 === '4' //false
+*/
+const greaterThanSign = 4 > 3;
+const greaterThanOrEqualTo = 4 >= 3;
+const lessThanSign = 4 < 3;
+const lessThanSignOrEqualTo = 4 <= 3;
+const comparisonSign = 4 == 4;
+const strictEqualitySign = 4 === 4;
+const notEqualToSign = 4 != 4;
+const strictInequalitySign = 4 !== '4';
+const notEqualToAndString = 4 != '4';
+const comparisonSignAndString = 4 == '4';
+const strictEqualityAndString = 4 === '4';
+
+console.log(greaterThanSign); //true
+console.log(greaterThanOrEqualTo); //true
+console.log(lessThanSign); //false
+console.log(lessThanSignOrEqualTo); //false
+console.log(comparisonSign); //true
+console.log(strictEqualitySign); //true
+console.log(notEqualToSign); //false
+console.log(strictInequalitySign); //true
+console.log(notEqualToAndString); //false
+console.log(comparisonSignAndString); //true
+console.log(strictEqualityAndString); //false
+
+//xii. Find the length of python and jargon and make a falsy comparison statement.
+const theComparisonBetweenPythonAndJargon = 'python'.length == 'jargon'.length;
```

Comment:
you forgot to build the falsy comparison statement. `'python'.length == 'jargon'.length` // true